### PR TITLE
Download the CSS file from eProsima/all-docs repository [8809]

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/13/)

File: docs/conf.py [hunk 1]
```diff
@@ -169,10 +226,9 @@
 html_static_path = ['_static']
 
 html_context = {
-        'css_files': [
-            '_static/css/fiware_readthedocs.css', #logo
-            ],
+        'css_files': select_css(html_build_dir),
         }
+print(html_context)
```
Comment (EduPonz) [inline docs/conf.py:231]: I'd remove this log and add one in select_css showing which style files are being used

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/14/)

File: docs/conf.py [hunk 2]
```diff
@@ -169,10 +230,9 @@
...
```
Comment (EduPonz) [inline docs/conf.py:235]: I'd remove this print

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/shapes_demo_docs_manual/15/)

**Pull request merged**